packages/report-server/src/app/addRoutesToApp.ts

```diff
@@ -38,7 +38,18 @@ export function addRoutesToApp(app: Express) {
   app.use(handleError);
 }
 
-const handleError = (err, req: ReportsRequest, res: Response) => {
-  const error = err.respond ? err : new InternalServerError(err);
+const handleError = (
+  err: RespondingError | Error,
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
```

Comment:
Basically, this wasn't working with just the 3 args. Copied the logic from `psss-server`

Reply:
Related: https://github.com/beyondessential/tupaia/pull/1686#discussion_r529935180